src/core/MenuItem/index.tsx

```diff
@@ -41,8 +92,19 @@ const MenuItem = forwardRef((props: MenuItemProps, _) => {
           className="primary-text"
           disabled={disabled}
         >
+          {sdsIcon && (
+            <StyledMenuItemIcon disabled={disabled}>
+              <Icon
+                {...sdsIconProps}
+                sdsType="static"
+                sdsIcon={sdsIcon}
+                sdsSize="s"
```

Comment:
just wanted to double check this is intended to be hardcoded as `s` and not accepting `xs` as an option

Reply:
MenuItems could only display an icon of size "s", according to the design file. I'll confirm with @clarsen-czi.

Reply:
As confirmed by @clarsen-czi, MenuItems only accept IconS.